public/ng-views/maps.html

```diff
@@ -629,47 +731,106 @@ <h3 class="noTopMargin noBottomMargin">
             ng-disabled="!showVectorIsosOn"
             ng-click="showVectorIsos(300*vectorIsos.val)"/>
         </span>
-        <i class="fa fa-plus-square"
+        <i class="fa fa-plus-square" 
             ng-click="vectorTimeVal_add(); showVectorIsos(300*vectorIsos.val)"></i>
       </span>
-      <span style="max-width:30%;display:inline-block" class="pull-right">
+      
+      <span style="max-width:30%" class="pull-right">
+      
+      <!-- RELOCATED to "TRAVEL TIME" ABOVE -->
+      <!-- <span style="max-width:30%;display:inline-block" class="pull-right">
           <h6 ng-click="toggleShowVectorIsos()">
             {{ vectorIsos.val*5 | minuteConverter }}
             <i class="fa fa-eye" ng-show="showVectorIsosOn"></i>
             <i class="fa fa-eye-slash" ng-show="!showVectorIsosOn"></i>
           </h6>
-      </span>
+      </span> -->
     </div>
-  </div>
 
-  <h4 ng-click="combosView=!combosView">
-    <span ng-show="!combos.sel">
-      Existing MBTA System
-    </span>
-    <span ng-show="combos.sel">
-      Scenario {{ combos.all[combos.sel].name }}
-      <span ng-show="scenarioCompare">
-        <span ng-show="combos.com">
-          Compared with {{ combos.all[combos.com].name }}
-        </span>
-        <span ng-show="!combos.com">
-          Compared with Baseline MBTA System
-        </span>
-      </span>
-    </span>
-      <i class="fa fa-angle-double-up pull-right" ng-show="!combosView"></i>
-      <i class="fa fa-angle-double-down pull-right" ng-show="combosView"></i>
-  </h4>
+  <h5 style="margin-bottom: -20px"> DESTINATIONS </h4> 
+    <br>
+    <label class="btn iconContainer" btn-radio="1"
+                  ng-model="" 
+                  ng-click="">
+           <i class="fa fa-briefcase fa-fw destinationIcon"></i>
```

Comment:
in your case it appears you have created a class for the sole purpose of increasing the font size (incorrectly, too, using em when you don't have anything nested correctly)... so in this situation 2 errors: 1. using em. 2. this could have been accomplished with one class within which these contents were nested. Read about it here: http://www.htmldog.com/guides/css/intermediate/grouping/